.github/workflows/tagged.yaml

```diff
@@ -14,15 +14,20 @@ jobs:
     steps:
     - name: Checkout
       uses: actions/checkout@v2
-    - name: Branch name
-      run: echo running on branch ${GITHUB_REF##*/}
     - name: Lint
       uses: actions-contrib/golangci-lint@master
       with:
         args: run
-    - name: Build the Docker images
+    - name: Export tag on pull request
+      run: | 
+        echo "::set-env name=TAG_NAME::${GITHUB_HEAD_REF##*/}"
```

Comment:
I stole that from https://github.com/actions/starter-workflows/issues/68#issuecomment-552074596

Reply:
If is ugly, why not a separate workflow with a matching "on" statement ?

Reply:
Mh. Because I named it "tagged" image. But I can also make a workflow "release" and one "pull_request". Maybe that's clearer then.